### ISSUE TITLE
If the bug is labelled as not in scope will it be considered accurate if the severity matches?

### ISSUE THREAD

**Issue by weijianwong** [nus-cs2103-AY2425S1/forum#829]
If the bug report is labelled as not in scope by the other team, but they did not change the severity of my bug report. Will this count towards my accuracy score even though it is labelled as not in scope?

**Comment by damithc** [comment]
Yes, the type/severity matches will increase your accuracy. `NotInScope` will have no impact on accuracy.
